worker/src/utils/shutdown.ts

```diff
@@ -0,0 +1,24 @@
+import logger from "../logger";
+
+import { evalJobCreator, evalJobExecutor } from "../queues/evalQueue";
+import { batchExportJobExecutor } from "../queues/batchExportQueue";
+import { flushIngestionQueueExecutor } from "../queues/ingestionFlushQueue";
+import { repeatQueueExecutor } from "../queues/repeatQueue";
+
+export const gracefulShutdown: NodeJS.SignalsListener = async (signal) => {
```

Comment:
Please check the logic on main branch
- we need to return status 500 on the health endpoint once we got sigterm
- we need to shut down the express server as well and stop accepting connections.